dependabot/update_repositories.py

```diff
@@ -116,25 +116,26 @@ def get_dependencies(module_name):
 # Update the gradle.properties file of each dependent module stored in the dependents list of modules with version updates
 def update_files(modules):
     for module in modules['modules']:
-        print("Updating dependents of " + module['name'])
+        print("Updating dependents of " + module['name'] + " to " + module['version'])
         for dependent in module['dependents']:
             # Fetch repository of the module
             repo = configure_github_repository(dependent)
             # Fetch gradle.properties file
             data = fetch_properties_file(repo, module['name'])
             # Update the gradle.properties file with version updates
-            updated_data, current_version, commit_flag = update_properties_file(data, module['name'], module['version'])
+            updated_data, current_version, commit_flag = update_properties_file(data, module['name'], module['version'], dependent)
             # If gradle.properties file is updated, commit changes and create PR
             if commit_flag:
                 try:
                     commit_changes(updated_data, current_version, repo, module['name'], module['version'])
                     create_pull_request(repo, current_version, module['name'], module['version'])
-                    print("Bump " + module['name'] + " version from " + current_version + " to " + module['version'])
+                    print("Bump version in " + dependent + " from " + current_version)
                     time.sleep(30)
                 except:
                     continue
             else:
-                print(module['name'] + " version is already the lastest version " + current_version)
+                if current_version:
```

Comment:
What does this condition do?

Reply:
If the current version in not set that means the version is not defined in the properties file. It is handled when updating the properties file. If the current version check is not done here, it will create two logs.